### PR TITLE
KEYCLOAK-18276 client content screen enhancement

### PR DESCRIPTION
**Pull request by cgeorgilakis**
https://github.com/keycloak/keycloak/pull/8092

### CODE REVIEW

**Comment by mposolda** [comment]
@cgeorgilakis Do you please have a chance to rebase this PR? Hopefully this should help to fix the conflicts.

@andymunro Do you please have a chance to review this PR?

File: server_admin/topics/clients/client-oidc.adoc [hunk 1]
```diff
@@ -97,6 +97,18 @@ for more information.
 
 If {project_name} needs to link to the client, this URL is used.
 
+*Logo URL*
+
+URL that references a logo for the Client application.
+
+*Policy URL*
+
```
Comment (andymunro) [inline server_admin/topics/clients/client-oidc.adoc]: remove "the" before "how"

File: server_admin/topics/clients/client-saml.adoc [hunk 2]
```diff
@@ -123,6 +123,18 @@ Valid Redirect URIs::
...
 Base URL::
   If {project_name} needs to link to the client, this URL would be used.
+  
+*Logo URL*
+
+URL that references a logo for the Client application.
+
+*Policy URL*
+
+URL that the Relying Party Client provides to the End-User to read about the how the profile data will be used.
```
Comment (andymunro) [inline server_admin/topics/clients/client-saml.adoc]: remove "the" before "how"

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @cgeorgilakis.  Just a couple of changes needed.

**Comment by cgeorgilakis** [comment]
@andymunro I have done the requested changes.
@mposolda client documentation format have been changed. I have placed the documentation in the new adoc files.

**Review by mposolda — APPROVED** [review]
@andymunro @cgeorgilakis Thanks to both of you for the review and for the requested changes and updating the documentation!

**Pull request merged**